### PR TITLE
Fix constructor bottom

### PR DESCRIPTION
**Pull request by M1ndForce**

### CODE REVIEW

**Review by jul4ajka — APPROVED** [review]

File: tests/constructor-bottom/constructor-bottom.helper.js [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+          }
+        }
+      }
+    `,
+    variables: { constructorElement: newDataConstructorBottom },
+  });
+  return createConstructorAgain.data.addConstructorBottom.message;
+};
+
+const updateConstructorB = async (wrongID, operations, addConstructor) => {
```
Comment (moran711) [inline tests/constructor-bottom/constructor-bottom.helper.js:98]: You already have `updateConstructor ` for this.

File: tests/constructor-bottom/constructor-bottom.helper.js [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+      id: constructorId,
+    },
+  });
+  return updateConstructor.data.updateConstructorBottom;
+};
+const createConstructorBottomAgain = async (
+  newDataConstructorBottom,
+  operations
+) => {
+  const createConstructorAgain = await operations.mutate({
```
Comment (moran711) [inline tests/constructor-bottom/constructor-bottom.helper.js:79]: You already have this func.

File: tests/constructor-bottom/constructor-bottom.helper.js [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+            image
+          }
+        }
+      }
+    `,
+  });
+  return allConstructorBottom.data.getAllConstructorBottom.items;
+};
+
+const createConstructorBottomQuery = async addConstructor => {
```
Comment (moran711) [inline tests/constructor-bottom/constructor-bottom.helper.js:203]: same

**Review by moran711 — APPROVED** [review]

File: tests/constructor-bottom/constructor-bottom.helper.js (hunk 1 above)
Comment (M1ndForce) [reply tests/constructor-bottom/constructor-bottom.helper.js:98]: Done. not necessary constructor deleted

**Comment by ivan-bonk** [comment]
merge after fix comments

File: tests/constructor-bottom/constructor-bottom.helper.js (hunk 2 above)
Comment (M1ndForce) [reply tests/constructor-bottom/constructor-bottom.helper.js:79]: done

File: tests/constructor-bottom/constructor-bottom.helper.js (hunk 3 above)
Comment (M1ndForce) [reply tests/constructor-bottom/constructor-bottom.helper.js:203]: done

**Review by ivan-bonk — APPROVED** [review]

**Pull request merged**